api/api.go

```diff
@@ -635,9 +636,10 @@ func OrbPublishByID(cl *graphql.Client, configPath string, orbID string, orbVers
 	}
 
 	query := `
-		mutation($config: String!, $orbId: UUID!, $version: String!) {
+		mutation($config: String!, $orbName: String!, $namespaceName: String!, $version: String!) {
```

Comment:
From the api-service GraphQL schema, it seemed these two were nullable (`version` definitely is non nullable). Should these be `String` and not `String!`?
Same question for https://github.com/CircleCI-Public/circleci-cli/pull/543/files#diff-bf98d5fab5bcfabded9769069e7318683f549ec91cff84c5f87a9ae3a3ca3d8dR1190